### PR TITLE
KEYCLOAK-12125 Update docs regarding SameSite changes

### PR DESCRIPTION
**Pull request by mhajas**

### CODE REVIEW

File: securing_apps/topics/oidc/javascript-adapter.adoc [hunk 1]
```diff
@@ -92,6 +92,7 @@ It has no other task than sending the received tokens to the main application an
 
 Please keep in mind that this page at the specified location must be provided by the application itself and is _not_ part of the JavaScript adapter!
 
+WARNING: Starting Chrome version 80 (released on February 2020) silent check-sso functionality will work only when SSL / TLS connection is configured on {project_name} side.
```
Comment (hmlnarik) [inline securing_apps/topics/oidc/javascript-adapter.adoc]:
```suggestion
WARNING: Starting Chrome version 80 (released on February 2020), silent `check-sso` functionality will work only when SSL / TLS connection is configured on {project_name} side.
```

File: securing_apps/topics/oidc/javascript-adapter.adoc [hunk 2]
```diff
@@ -154,6 +156,11 @@ This feature can be disabled by setting `checkLoginIframe: false` in the options
 You should not rely on looking at this cookie directly. Its format can change and it's also associated with the URL of the {project_name} server, not
 your application.
 
+WARNING: Starting Chrome version 80 (released on February 2020), status iframe will only be able to see the special
+cookie over SSL / TLS connection configured on {project_name} side. Using insecure connection, may lead to redirecting
```
Comment (hmlnarik) [inline securing_apps/topics/oidc/javascript-adapter.adoc]:
```suggestion
cookie over SSL / TLS connection configured on {project_name} side. Using insecure connection may lead to redirecting
```

File: securing_apps/topics/oidc/javascript-adapter.adoc [hunk 3]
```diff
@@ -92,6 +92,8 @@ It has no other task than sending the received tokens to the main application an
 
 Please keep in mind that this page at the specified location must be provided by the application itself and is _not_ part of the JavaScript adapter!
 
+WARNING: Starting Chrome version 80 (released on February 2020), silent `check-sso` functionality will work only when
```
Comment (hmlnarik) [inline securing_apps/topics/oidc/javascript-adapter.adoc]:
```suggestion
WARNING: Starting with Chrome version 80 (released on February 2020), silent `check-sso` functionality will work only when
```

File: release_notes/topics/9_0_0.adoc [hunk 4]
```diff
@@ -14,6 +14,13 @@ A new built-in vault provider that reads secrets from a keystore-backed Elytron
 extension. The creation and management of the credential store is handled by Elytron using either the `elytron` subsystem or the
 `elytron-tool.sh` script.
 
+== SameSite Cookie changes with upcoming Google Chrome update
```
Comment (hmlnarik) [inline release_notes/topics/9_0_0.adoc]:
```suggestion
== SameSite cookie changes with upcoming Google Chrome update
```

File: release_notes/topics/9_0_0.adoc [hunk 5]
```diff
@@ -14,6 +14,13 @@ A new built-in vault provider that reads secrets from a keystore-backed Elytron
...
+
+Google chrome starting version 80, will change its default value for SameSite cookie argument to `Lax`. Therefore we
```
Comment (hmlnarik) [inline release_notes/topics/9_0_0.adoc]:
```suggestion
Google Chrome starting with version 80 will change the default value for `SameSite` cookie parameter to `Lax`. Therefore
```

File: release_notes/topics/9_0_0.adoc [hunk 6]
```diff
@@ -14,6 +14,13 @@ A new built-in vault provider that reads secrets from a keystore-backed Elytron
...
+needed to change some of our cookies (especially those which are used within our Javascript adapter for checking session
```
Comment (hmlnarik) [inline release_notes/topics/9_0_0.adoc]:
```suggestion
changes were required to several {project_name} cookies (especially those which are used within our Javascript adapter for checking session
```

File: release_notes/topics/9_0_0.adoc [hunk 7]
```diff
@@ -14,6 +14,13 @@ A new built-in vault provider that reads secrets from a keystore-backed Elytron
...
+status using the iframe) to have SameSite argument set to None which requires also setting the argument Secure. This
```
Comment (hmlnarik) [inline release_notes/topics/9_0_0.adoc]:
```suggestion
status using the iframe) to set SameSite parameter to `None`. Please note that this settings also requires setting the Secure parameter, hence
```

File: release_notes/topics/9_0_0.adoc [hunk 8]
```diff
@@ -14,6 +14,13 @@ A new built-in vault provider that reads secrets from a keystore-backed Elytron
...
+means starting with version 9.0.0 our Javascript adapter will be fully functional only using TLS on {project_name} side.
```
Comment (hmlnarik) [inline release_notes/topics/9_0_0.adoc]:
```suggestion
starting with this version, our Javascript adapter will only be fully functional when using SSL / TLS connection on {project_name} side.
```

File: release_notes/topics/9_0_0.adoc [hunk 9]
```diff
@@ -14,6 +14,14 @@ A new built-in vault provider that reads secrets from a keystore-backed Elytron
 extension. The creation and management of the credential store is handled by Elytron using either the `elytron` subsystem or the
 `elytron-tool.sh` script.
 
+== SameSite cookie changes with upcoming Google Chrome update
+
+Google Chrome starting with version 80, will change the default value for `SameSite` cookie parameter to `Lax`.
```
Comment (hmlnarik) [inline release_notes/topics/9_0_0.adoc]:
```suggestion
Starting with version 80, Google Chrome will change the default value for `SameSite` cookie parameter to `Lax`.
```

**Review by hmlnarik — CHANGES_REQUESTED** [review]
Thanks @mhajas ! Can you please check Travis? The failure seems relevant

**Review by hmlnarik — DISMISSED** [review]

**Review by abstractj — DISMISSED** [review]

**Comment by stianst** [comment]
Wonder if this should be listed in 8.0.2 changes instead of 9.0.0, as it will be released in 8.0.2.

**Review by stianst — COMMENTED** [review]
See previous comment

**Comment by bigred8982** [comment]
@stianst Is this change expected to make it into 8.0.2? 

I don't see an issue related to this in the 8.0.2 release - https://issues.redhat.com/projects/KEYCLOAK/versions/12343302

**Review by hmlnarik — DISMISSED** [review]

**Comment by hmlnarik** [comment]
The travis failures are irrelevant to this PR.

File: release_notes/topics/8_0_2.adoc [hunk 10]
```diff
@@ -0,0 +1,9 @@
+= Highlights
+
+== SameSite cookie changes with upcoming Google Chrome update
+
+Starting with version 80, Google Chrome will change the default value for `SameSite` cookie parameter to `Lax`.
```
Comment (andymunro) [inline release_notes/topics/8_0_2.adoc]: Add "the" before 'SameSIte'

File: release_notes/topics/8_0_2.adoc [hunk 11]
```diff
@@ -0,0 +1,9 @@
...
+Therefore changes were required to several {project_name} cookies (especially those which are used within our
```
Comment (andymunro) [inline release_notes/topics/8_0_2.adoc]: Add a comma after Therefore

Change "our" to "the"

File: release_notes/topics/8_0_2.adoc [hunk 12]
```diff
@@ -0,0 +1,9 @@
...
+Javascript adapter for checking session status using the iframe) to set `SameSite` parameter to `None`. Please note
```
Comment (andymunro) [inline release_notes/topics/8_0_2.adoc]: Add "the" before session status

Add "the" before 'SameSIte'

File: release_notes/topics/8_0_2.adoc [hunk 13]
```diff
@@ -0,0 +1,9 @@
...
+that this settings also requires setting the `Secure` parameter, hence starting with this version, our Javascript
```
Comment (andymunro) [inline release_notes/topics/8_0_2.adoc]: Change "our" to "the"

File: release_notes/topics/8_0_2.adoc [hunk 14]
```diff
@@ -0,0 +1,9 @@
...
+adapter will only be fully functional when using the SSL / TLS connection on {project_name} side.
```
Comment (andymunro) [inline release_notes/topics/8_0_2.adoc]: add "the" before {project_name}

File: securing_apps/topics/oidc/javascript-adapter.adoc [hunk 15]
```diff
@@ -92,6 +92,8 @@ It has no other task than sending the received tokens to the main application an
 
 Please keep in mind that this page at the specified location must be provided by the application itself and is _not_ part of the JavaScript adapter!
 
+WARNING: Starting with Chrome version 80 (released on February 2020), silent `check-sso` functionality will work only
+when the SSL / TLS connection is configured on {project_name} side.
```
Comment (andymunro) [inline securing_apps/topics/oidc/javascript-adapter.adoc]: add "the" before {project_name}

File: securing_apps/topics/oidc/javascript-adapter.adoc [hunk 16]
```diff
@@ -154,6 +156,11 @@ This feature can be disabled by setting `checkLoginIframe: false` in the options
 You should not rely on looking at this cookie directly. Its format can change and it's also associated with the URL of the {project_name} server, not
 your application.
 
+WARNING: Starting with Chrome version 80 (released on February 2020), status iframe will only be able to see the special
+cookie over the SSL / TLS connection configured on {project_name} side. Using insecure connection may lead to redirecting
```
Comment (andymunro) [inline securing_apps/topics/oidc/javascript-adapter.adoc]: add "the" before {project_name} 

Add "an" before "insecure."

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks good apart from the changes I suggested.

**Comment by mhajas** [comment]
Thank you @andymunro, I updated the PR.

**Review by abstractj — APPROVED** [review]
@stianst @hmlnarik LGTM

**Pull request merged**